src/components/Create/components/RewardsList/AddEditRewardModal.tsx

```diff
@@ -236,19 +282,108 @@ export const AddEditRewardModal = ({
             )}
           </Space>
         </Form.Item>
-        <Form.Item
-          name="externalUrl"
-          label={t`External link`}
-          rules={[
-            inputIsValidUrlRule({
-              label: t`External link`,
-              prefix: 'https://',
-              validateTrigger: 'onCreate',
-            }),
-          ]}
+        <CreateCollapse
+          activeKey={advancedOptionsOpen ? ['0'] : []}
+          onChange={onCollapseChanged}
         >
-          <PrefixedInput prefix="https://" />
-        </Form.Item>
+          <CreateCollapse.Panel
+            header={<OptionalHeader header={t`Advanced options`} />}
+            key={0}
+            hideDivider
+          >
+            <Form.Item
+              extra={
+                <span className="text-sm">
+                  <Trans>Set aside a percentage of freshly minted NFTs.</Trans>
+                </span>
+              }
+            >
+              <JuiceSwitch
+                value={isReservingNfts}
+                onChange={setIsReservingNfts}
+                label={
+                  <span className="flex items-center gap-2">
+                    <Trans>Reserve NFTs</Trans>
+                    <Tooltip
+                      className="cursor-help text-grey-500 dark:text-grey-300"
+                      title={t`If Reserved NFTs are set, the first reserved NFT from the tier will be distributable once the tier receives its first mint.`}
+                    >
+                      <QuestionCircleOutlined />
+                    </Tooltip>
+                  </span>
+                }
+              />
+            </Form.Item>
+            {isReservingNfts && (
+              <>
+                <span className="mb-4 flex flex-col gap-2 whitespace-nowrap md:flex-row md:items-center">
+                  <Trans>Reserve 1 NFT of every</Trans>
+                  <div className="max-w-[78px]">
+                    <Form.Item
+                      className="mb-0"
+                      name="nftReservedRate"
+                      rules={[
+                        inputMustExistRule({ label: t`Reserved rate` }),
+                        inputIsIntegerRule({
+                          stringOkay: true,
+                        }),
+                      ]}
+                    >
+                      <FormattedNumberInput min={2} placeholder="0" />
```

Comment:
<img width="489" alt="Screen Shot 2023-01-24 at 2 39 54 pm" src="https://user-images.githubusercontent.com/96150256/214213504-7c7e75f4-6632-4539-ada9-bee05f9631af.png">

Error state is a bit off here. Maybe just 'Required'? And would be great to have the input field not jump up like it is but not blocking

Reply:
We can definitely change to just required. The jump up is default behavior of antd unfortunately :(